public/js/whydPlayer.js

```diff
@@ -496,19 +496,31 @@ function WhydPlayer() {
   // handlers for events coming from Playem
 
   var playemEventHandlers = {
-    onError: function(e) {
+    onError: function (e) {
       if (currentTrack.metadata.logData) logTrackPlay();
 
+      // will try to fallback failing tracks to deezer player
+      //Detect user agent for electron specific message
+      var userAgent = navigator.userAgent.toLowerCase();
+      var isElectronApp = false;
+      var failedTrackMessage;
+      if (userAgent.indexOf(' electron/') > -1) {
+        isElectronApp = true;
+        failedTrackMessage = 'Oops, we could not play this track...'
+      } else {
+        failedTrackMessage = 'Oops, we could not play' +
+          ' <a href="' +
+          currentTrack.metadata.url +
+          ' target="_blank">this track</a>...' +
+          ' Please try with <a href="https://openwhyd.org/download"' +
+          ' target="_blank">Openwhyd Desktop App</a> 👌'
+      }
+
       // will try to fallback failing tracks to deezer player
```

Comment:
Also, this is not at all critical, but feel free to remove that old comment that does not correspond to the code anymore.

Reply:
got it :+1: